docs/README.md

```diff
@@ -1,129 +0,0 @@
-# SQLCommenter
```

Comment:
Any reason to remove this file ? Its the start page of sqlcommenter doc

Reply:
Added again

Reply:
Added again